### PR TITLE
Remove call for beta testers.

### LINKED ISSUE #450: Remove call(s) for beta pilot testers
Now that the lesson is stable we can remove the call(s) for beta pilot testers.

### PR DESCRIPTION
**Pull request by carschno**
Fixes #450

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -41,9 +41,9 @@ This indicates that the maintainers will welcome a pull request fixing this issu
...
 We would be very grateful if you can provide us with feedback on this lesson. You can read more about hosting a lesson pilot for an incubator lesson [here](https://docs.carpentries.org/topic_folders/lesson_development/lesson_pilots.html).
 
-You can notify us that you plan to teach this lesson by creating an issue in this repository (and labeling it with beta) or posting a message in the carpentries Slack Machine Learning channel. Please note the questions below to get an indication of the sort of feedback we expect.
+You can notify us that you plan to teach this lesson by creating an issue in this repository or posting a message in the carpentries Slack Machine Learning channel. Please note the questions below to get an indication of the sort of feedback we expect.
 
-After the workshop, lease create an issue (or comment on the issue you created before teaching) with general feedback on teaching the lesson, and label it with beta. As a template, you can use the following questions:
+After the workshop, lease create an issue (or comment on the issue you created before teaching) with general feedback on teaching the lesson. As a template, you can use the following questions:
```
Comment (svenvanderburg) [inline README.md:46]:
```suggestion
After the workshop, please create an issue (or comment on the issue you created before teaching) with general feedback on teaching the lesson. As a template, you can use the following questions:
```

File: index.md [hunk 2]
```diff
@@ -20,21 +20,6 @@ Learners are expected to have the following knowledge:
 
 ::: instructor
```
Comment (svenvanderburg) [inline index.md:21]: Looks like this opening statement should also be removed? (or needs a closing if you like my comment below)

File: index.md [hunk 3]
```diff
@@ -20,21 +20,6 @@ Learners are expected to have the following knowledge:
...
 
-## Looking for Beta Testers!
```
Comment (svenvanderburg) [inline index.md:23]: I would suggest to instead replace this with:

## We can help you out with teaching this lesson!
Do you want to teach this lesson? Feel free to reach out to us with any questions that you have. Just [open a new issue](https://github.com/carpentries-incubator/deep-learning-intro/issues/new/choose). We also value any feedback on the lesson!

**Review by svenvanderburg — APPROVED** [review]
I left a few comments. Feel free to merge afterwards.

File: index.md (hunk 3 above)
Comment (carschno) [reply index.md:23]: That would seem a bit redundant with the ["Teaching this lesson?" section in the README](https://github.com/carpentries-incubator/deep-learning-intro/blob/fddec665ef5ae5885ad411246f2eaa93b972ea0e/README.md#L41). Maybe a link there would be better to keep the information in sync?

File: index.md (hunk 2 above)
Comment (carschno) [reply index.md:21]: I understand that this line now opens the "Breaks and Schedule" div.

**Comment by carschno** [comment]
Closes #504

File: index.md (hunk 3 above)
Comment (carschno) [reply index.md:23]: @svenvanderburg How about [this](https://github.com/carpentries-incubator/deep-learning-intro/blob/cd26b43ed0728a4dc83d31dda2e2634f32e7e017/index.md#we-can-help-you-out-with-teaching-this-lesson)?

Comment (svenvanderburg) [reply index.md:23]: Good point! I like your solution :)

**Review by svenvanderburg — APPROVED** [review]

**Pull request merged**